File: C/Lista_01(Estrutura_Sequencial)/Retangulo.c
#include <stdio.h>
#include <math.h>

int main () {

    double base, altura;

    printf("Base do retangulo: ");
    scanf("%lf", &base);

    printf("Altura do retangulo: ");
    scanf("%lf", &altura);

    double area = base * altura;

    double perimetro = 2*base + 2*altura;

    double diagonal = sqrt(pow(base,2)+pow(altura,2));

    printf("AREA = %.4lf\n", area);
    printf("PERIMETRO = %.4lf\n", perimetro);
    printf("DIAGONAL = %.4lf", diagonal);

}

File: Java/Lista_01(Estrutura_Sequencial)/Retangulo.java
import java.util.Scanner;
import java.util.Locale;

public class Retangulo {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner s = new Scanner(System.in);
		
		System.out.print("Base do retangulo: ");
		double base = s.nextDouble();
		
		System.out.print("Altura do retangulo: ");
		double altura = s.nextDouble();
		
		double area = base * altura;
		double perimetro = 2*base + 2*altura;
		double diagonal = Math.sqrt(Math.pow(base, 2)+Math.pow(altura, 2));
		
		System.out.printf("Area = %.4f\n", area);
		System.out.printf("Perimetro = %.4f\n", perimetro);
		System.out.printf("Diagonal = %.4f", diagonal);
		
		s.close();

	}

}
